documentation/releaseNotes/releaseNotes.v7.0.0.md

```diff
@@ -0,0 +1,7 @@
+Today we are releasing the 7.0.0 build of the `dotnet monitor` tool. This release includes:
+
+- Fixed an issue where AspNet* collection rules could stop functioning after collecting a trace with the Http profile. The fix was made in [dotnet/diagnostics#3425](https://github.com/dotnet/diagnostics/pull/3425). ([#2764](https://github.com/dotnet/dotnet-monitor/pull/2764))
+- Change the default Garbage Collector mode to Workstation, except when running inside one of the official `dotnet monitor` docker images with more than 1 logical CPU core available. Learn more [here](https://github.com/dotnet/dotnet-monitor/blob/main/documentation/configuration.md#garbage-collector-mode). ([#2737](https://github.com/dotnet/dotnet-monitor/pull/2737))
+
+\*🔬 **_indicates an experimental feature_** \
```

Comment:
Let's remove these two so as not to lead anyone to believe there are new experimental features or breaking changes.